src/types.db

```diff
@@ -192,6 +192,27 @@ snr			value:GAUGE:0:U
 spam_check		value:GAUGE:0:U
 spam_score		value:GAUGE:U:U
 spl			value:GAUGE:U:U
+stenographer_aged_files value:COUNTER:0:U 
```

Comment:
This needs some work: You need to have one "type" for each data type you're handling, so good types would be "bytes", "blocks", "queries", … I'll leave some suggestions below.